ocsp.go

```diff
@@ -815,28 +815,30 @@ func extractOCSPCacheResponseValue(cacheValue []interface{}, subject, issuer *x5
 // writeOCSPCacheFile writes a OCSP Response cache file. This is called if all revocation status is success.
 // lock file is used to mitigate race condition with other process.
 func writeOCSPCacheFile() {
+	if strings.EqualFold(os.Getenv(cacheServerEnabledEnv), "false") {
+		logger.Infof("skipping writing OCSP Cache.")
```

Comment:
I would make this a little more prominent, because this is going to be the only line that would signify that the cache is turned off. Also turning the cache off can have major impacts so this should be made a little more prominent.

Reply:
Noted.